### ISSUE TITLE
Syntax error when running udocker install

### ISSUE THREAD

**Issue by Smyja** [indigo-dc/udocker#374]
I get Syntax error when i try to run ```udocker install```
```
Traceback (most recent call last):
  File "/home/maro/udocker/udocker", line 28, in <module>
    from udocker.umain import UMain
  File "/home/maro/udocker/../udocker/umain.py", line 9, in <module>
    from udocker.container.localrepo import LocalRepository
  File "/home/maro/udocker/../udocker/container/localrepo.py", line 13, in <module>
    from udocker.utils.fileutil import FileUtil
  File "/home/maro/udocker/../udocker/utils/fileutil.py", line 12, in <module>
    from udocker.helper.unique import Unique
  File "/home/maro/udocker/../udocker/helper/unique.py", line 40
    return f"{self._rnd(8)}-{self._rnd(4)}-{self._rnd(4)}-{self._rnd(4)}-{self._rnd(12)}"
                                                                                        ^
SyntaxError: invalid syntax
```
I am using Ubuntu 20.04.4 LTS
I followed the installation guide [here](https://indigo-dc.github.io/udocker/installation_manual.html)

and ran 
```
curl -L https://github.com/indigo-dc/udocker/releases/download/v1.3.4/udocker-1.3.4.tar.gz \
  > udocker-1.3.4.tar.gz
tar zxvf udocker-1.3.4.tar.gz
export PATH=`pwd`/udocker:$PATH
```

**Comment by jorge-lip** [comment]
Please use python >= 3.6. We will be releasing soon a new release
with fixes including this one. 


On Mon, 2022-10-10 at 04:12 -0700, Smyja wrote:
> I get Syntax error when i try to run udocker install
> Traceback (most recent call last):
>   File "/home/maro/udocker/udocker", line 28, in <module>
>     from udocker.umain import UMain
>   File "/home/maro/udocker/../udocker/umain.py", line 9, in <module>
>     from udocker.container.localrepo import LocalRepository
>   File "/home/maro/udocker/../udocker/container/localrepo.py", line
> 13, in <module>
>     from udocker.utils.fileutil import FileUtil
>   File "/home/maro/udocker/../udocker/utils/fileutil.py", line 12, in
> <module>
>     from udocker.helper.unique import Unique
>   File "/home/maro/udocker/../udocker/helper/unique.py", line 40
>     return f"{self._rnd(8)}-{self._rnd(4)}-{self._rnd(4)}-
> {self._rnd(4)}-{self._rnd(12)}"
>                                                                      
>                    ^
> SyntaxError: invalid syntax
> I am using Ubuntu 20.04.4 LTS
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this
> thread.Message ID: ***@***.***>

-- 
LIP - Laboratório de Instrumentação e Física Experimental de Partículas

**Comment by mariojmdavid** [comment]
solved in 1.3.5
